# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/webrender/src/debug_render.rs
# Commit: 86db847cfe57
# Full Hash: 86db847cfe570dba25289ffbe52aaaaecf76bc70
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2020-12-18 04:07:40
# Regressor Bug: 1682365
# File Overlap Count: 1
# Description:
#   Bug 1682365 - Pt 2 - Remove texture array usage from render task graph. r=nical
#   
#   With this change, all color/alpha intermediate surfaces are individual
#   textures, rather than texture arrays.
#   
# ==============================================================================

diff -r c3495de132cd -r 86db847cfe57 gfx/wr/webrender/src/debug_render.rs
--- a/gfx/wr/webrender/src/debug_render.rs	Thu Dec 17 19:53:54 2020 +0000
+++ b/gfx/wr/webrender/src/debug_render.rs	Thu Dec 17 19:54:02 2020 +0000
@@ -140,7 +140,7 @@
         let tri_vao = device.create_vao(&DESC_COLOR);
 
         let font_texture = device.create_texture(
-            ImageBufferKind::Texture2DArray,
+            ImageBufferKind::Texture2D,
             ImageFormat::R8,
             debug_font_data::BMP_WIDTH,
             debug_font_data::BMP_HEIGHT,